xml/art_sle_ha_pmremote.xml

```diff
@@ -144,37 +158,50 @@
        A virtual machine that runs the &pmremote; daemon.
        A guest node is configured using the <option>remote-node</option>
        metadata option of a resource agent such as
-       <systemitem>ocf:pacemaker:VirtualDomain</systemitem>.
+       <systemitem>ocf:pacemaker:VirtualDomain</systemitem> (see use case&nbsp;2
+       in <xref linkend="tab.ha.pmremote.usecases"/>)
```

Comment:
A guest node is created using a resource agent such as ocf:heartbeat:VirtualDomain with the "remote-node"  meta attribute.